streams/jp.m3u

```diff
@@ -46,14 +42,14 @@ http://51.79.193.108:8080/hlsok/j00050/index.m3u8?type=jp
 http://cgntv-glive.ofsdelivery.net/live/cgntv_jp/playlist.m3u8
 #EXTINF:-1 tvg-id="FujiTV.jp" status="online",Fuji TV (720p) [Not 24/7]
 http://51.79.193.108:8080/hls/j00010/index.m3u8?type=jp
-#EXTINF:-1 tvg-id="FujiTV.jp" status="error",Fuji TV (540p) [Not 24/7]
-https://fujitv1.mov3.co/hls/fujitv.m3u8
+#EXTINF:-1 tvg-id="FujiTV.jp" status="",Fuji TV (540p) [Not 24/7]
+https://fujitv2.mov3.co/hls/fujitv.m3u8
 #EXTINF:-1 tvg-id="GAORA.jp" status="online",GAORA (540p) [Not 24/7]
 http://51.79.193.108:8080/hls/j00045/index.m3u8?type=jp
 #EXTINF:-1 tvg-id="GAORA.jp" status="online",GAORA (540p) [Not 24/7]
 http://51.79.193.108:8080/hlsok/j00045/index.m3u8?type=jp
-#EXTINF:-1 tvg-id="GSTV.jp" status="error",GSTV (720p)
-https://gemstv.wide-stream.net/gemstv01/smil:gemstv01.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="GSTV.jp" status="online",GSTV (720p)
+https://gstv-tnz-gsmediastreaming.preview-jpea.channel.media.azure.net/dfd06b62-e9d1-4a7f-bcbb-89d2ecbc82ee/preview.ism/manifest(format=mpd-time-csf,audio-only=false)
```

Comment:
<img width="756" alt="image" src="https://user-images.githubusercontent.com/7253922/162543992-c9242e94-7ed2-401f-9542-505111c7ef37.png">

Reply:
![manifest(format=m3u8-aapl,audio-only=false)_snapshot_00 17 442](https://user-images.githubusercontent.com/30985701/162546963-4792019b-54ad-4dc3-a4d2-a404513092fa.jpg)

Works badly indeed, only seems to load correctly on browser